orocos_kdl/src/chainhdsolver_vereshchagin_doc.md

```diff
@@ -0,0 +1,206 @@
+
```

Comment:
This is very nice to have, but it is the best if this also is shown in documentation generated by Doxygen. So please check how that can be included nicely. 

Right now I have to idea it is better to move this to the header.

Either way, please check the generated documentation.

Reply:
I see. Thanks for the feedback! 

I do not have so much experience with generating Doxygen docs, but I will for sure look into it.
Regarding the idea of moving doc to the header: the documentation is extensive but it is necessary due to solver's complexity, thus, I think it might be too large for the header. 
 
Another idea would be to move this doc file to "orocos_kdl/doc" and reference it in solver's source and header files.

Reply:
Maybe @smits can give us more ideas for the location of and generating Doxygen documentation since he has a lot of experience with documenting the KDL.

Reply:
I have generated the docs locally, but this entire file does not show. I stay with my opinion this should be included in the header.

Reply:
No problem. I will do that as well.

Reply:
The documentation is now moved to the header file. I have generated the Doxygen HTML file on my machine to see the results. Try the document on your browser:  https://drive.google.com/file/d/1UvEQMq4j9Se5z7wX7w5siyjMopU068oM/view?usp=sharing to see if it is ok.

Btw, what should I do with the already existing markdown file (chainhdsolver_vereshchagin_doc.md): should I delete it from the pull request or move it under /orocos_kdl/doc directory?